iota-client/Cargo.toml

```
@@ -17,7 +17,7 @@ bee-transaction = { git = "https://github.com/Alex6323/bee-p.git" }
 bee-ternary = { version = "0.3.1-alpha", git = "https://github.com/iotaledger/bee.git", features = ["serde1"] }
 bee-signing = { git = "https://github.com/iotaledger/bee.git" }
 bee-crypto = { git = "https://github.com/iotaledger/bee.git" }
-reqwest = { version = "0.10.6", features = ["json"] }
+reqwest = { version = "0.10.6", features = ["json"], default-features = false }
```

It still have to choose one of tls library. The most sane choice I think is rustls. But it might bloat the size a bit.

```suggestion
reqwest = { version = "0.10.6", features = ["json", "rustls-tls"], default-features = false }
```

---

yeah I see a jump from 5.5MB to 7.1MB when using rustls-tls, so we should use it only when compiling to Android (at least for now).